qa/pull-tester/rpc-tests.sh

```diff
@@ -20,7 +20,6 @@ testScripts=(
     'wallet_protectcoinbase.py'
     'wallet_shieldcoinbase_sprout.py'
     'wallet_shieldcoinbase_sapling.py'
-    'wallet_listreceived.py'
```

Comment:
There are pros and cons to consolidating these tests. We are able to take advantage of the fact that the setup is the same and maybe we should always test these two rpcs together. In some situations it could also be nice to have them decoupled. I could go either way. If we remove this test, I wonder if we should also remove `wallet_changeindicator.py`?